index.html

```diff
@@ -14,7 +14,7 @@
     <meta charset="UTF-8">
```

Comment:
[ OPTIONAL ]
The report should include a check about page titles.
The report should include a check about image text alternatives (img alt="alt text ").
The report should include a check about text headings.
The report should include a check about color contrast.
The report should include a check about resizing.
The report should include a check about interaction.
The report should include a check about moving content.
The report should include a check about multimedia.
The report should include a check about the basic structure of the page.

Please include a statement about every point checked in the result section of the report as described below:

If everything was ok, please make a statement about that (e.g. "Page titles - no issues found").
If issues were found, please provide a link to the corresponding line of code in the Pull Request which you change to follow the accessibility rule.

Please go through the requirements of this milestone from [this](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md) link.